firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -44,6 +47,29 @@
   private static final int EVENT_THREAD_IMPORTANCE = 4;
   private static final int MAX_CHAINED_EXCEPTION_DEPTH = 8;
 
+  private static final int DEFAULT_MAX_EVENTS_TO_KEEP = 8;
+  private static final int DEFAULT_MAX_REPORTS_TO_KEEP = 4;
+
+  public static SessionReportingCoordinator create(
```

Comment:
Should the other constructor become private?

Reply:
No, but package-private for testing, yes.

Reply:
The constructor is currently package-private, so not taking any action here.